main.go

```
@@ -62,19 +168,38 @@
 	setupLogger()
 }
 
 func main() {
 	flag.Parse()
 	database.SetDBInfo(cfg)
-	web.ListenAndServe(cfg)
+	if dev {
+		version = "dev"
+		debugOutput()
+	}
+	web.ListenAndServe(cfg, version)
```

golangci-lint (1.62.2)

Error return value of `web.ListenAndServe` is not checked

(errcheck)